Assets/Scripts/Core/GameLevelInitializer.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using Core.Services.Updater;
+using InputReader;
+using Player;
+using UnityEngine;
+
+namespace Core
+{
+    public class GameLevelInitializer : MonoBehaviour
+    {
+        [SerializeField] private PlayerEntity _playerEntity;
+        [SerializeField] private GameUIInputView _gameUIInputView;
+
+        private ExternalDevicesInputReader _externalDevicesInputReader;
+        private PlayerSystem _playerSystem;
+        private ProjectUpdater _projectUpdater;
```

Comment:
Field '_projectUpdater' is assigned but its value is never used

Reply:
that is for future :)